pallets/exchange-rate/Cargo.toml

```diff
@@ -32,55 +32,46 @@ serde = { version = '1.0.101', optional = true, features = ['derive'] }
 [dependencies.pallet-balances]
```

Comment:
could we please change all the Cargo.toml dependencies so they are inline like we've done in our standalone chain code.

i.e. this would be listed under `[dependencies]` and `[dependencies.pallet-balances]` wouldn't be required, and instead written 
```
pallet-balances = { git = 'https://github.com/paritytech/substrate', branch = 'polkadot-v0.9.9', default-features = false }
```
also, please use single quotes instead of double quotes

Reply:
added to https://github.com/DataHighway-DHX/node/issues/231